src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -728,6 +728,17 @@ def _handle_group_actions(
 
         return self.respond(response)
 
+    def _send_blocks(self, response: SlackBlock, response_url: str, text: str | None = None) -> str:
+        json_blocks = orjson.dumps(response.get("blocks")).decode()
+        webhook_client = WebhookClient(response_url)
+        webhook_client.send(
+            blocks=json_blocks,  # type: ignore[arg-type]
+            text=text,
+            delete_original=False,
+            replace_original=True,
+        )
```

Comment:
Sending a `str` as the `blocks` argument just plain contradicts the library's type annotation (`Optional[Sequence[Union[Dict[str, Any], Block]]]`). I'm guessing that we're taking advantage of an (undocumented?) feature where a string value of valid JSON gets parsed into the equivalent `Block`s.

It sure looks like we could just send `response.get("blocks")` as the value of `blocks`, but I hesitate to change that without knowing why it was dumped to a string in the first place (principle of Chesterson's Fence). Was it solely so it could be passed to the logger on line 714?

I'd like to attempt that in a follow-up PR (and I'd be comfortable shipping it if no tests break), but for now this is refactoring is behaviorally equivalent (I only extracted the new helper method so that we would only need the type-ignore comment in one place).